src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -199,34 +199,19 @@ object EasyIngest {
       request = (datastreamId, sdo.listFiles.find(_.getName == dsSpec.contentFile)) match {
         case ("EMD", Some(file)) =>
           // Note that this would change the ingested file's checksum, but it is only for the EMD datastream, which has no checksum
-          return executeAddRequestWithReplacement(request, file, PLACEHOLDER_FOR_DMO_ID, pidDictionary(sdo.getName))
+          request.content(replacePlaceHolder(file, PLACEHOLDER_FOR_DMO_ID, pidDictionary(sdo.getName)))
```

Comment:
See [this closing mechanism that I developed in the License Creator](https://github.com/rvanheest-DANS-KNAW/easy-license-creator/blob/master/src/main/scala/nl/knaw/dans/easy/license/package.scala#L205). Note, this is again something interesting for the utils lib...

For usages, see:
* [TemplateResolver](https://github.com/rvanheest-DANS-KNAW/easy-license-creator/blob/master/src/main/scala/nl/knaw/dans/easy/license/TemplateResolver.scala#L61)
* [LicenseCreator (2x)](https://github.com/rvanheest-DANS-KNAW/easy-license-creator/blob/master/src/main/scala/nl/knaw/dans/easy/license/LicenseCreator.scala#L33)
* [Fedora](https://github.com/rvanheest-DANS-KNAW/easy-license-creator/blob/master/src/main/scala/nl/knaw/dans/easy/license/Fedora.scala) (`Observable` variant)
* [Command](https://github.com/rvanheest-DANS-KNAW/easy-license-creator/blob/master/src/main/scala/nl/knaw/dans/easy/license/Command.scala#L36) (`Observable` variant)